Makefile

```diff
@@ -79,6 +89,10 @@ set_env_class_stage:
 	$(eval export ENABLE_AUTO_DEPLOY=true)
 	$(eval export DISABLE_AUTODELETE=1)
 	$(eval export PAAS_CF_TAG_FILTER=stage-*)
+	$(eval export ROOT_DNS_ZONE=cloudpipeline.digital)
+	$(eval export APPS_ROOT_DNS_ZONE=cloudpipelineapps.digital)
+	$(eval export ENV_DNS_ZONE=${AWS_ACCOUNT}.${ROOT_DNS_ZONE})
```

Comment:
I am also unsure about `ENV_TYPE` because it doesn't describe the usage (DNS prefix?). It looks like it's only used to construct `ENV_APPS_DNS_ZONE`, in which case do we need a variable at all?